qiskit/algorithms/minimum_eigensolvers/diagonal_estimator.py

```diff
@@ -171,8 +171,9 @@ def aggregate(measurements):
             accumulated_percent = 0  # once alpha is reached, stop
             cvar = 0
             for probability, value in sorted_measurements:
-                cvar += value * max(probability, alpha - accumulated_percent)
+                cvar += value * min(probability, alpha - accumulated_percent)
                 accumulated_percent += probability
+                print(value, probability, accumulated_percent, cvar)
```

Comment:
```suggestion
```

Reply:
If something like this is useful we could always have a logging statement here instead that does similar

Reply:
whoops -- I thought I removed all statements!

Reply:
@woodsp-ibm That was for debugging purposes only 🙂